---
title: IBMQProviderError
description: API reference for qiskit.providers.ibmq.IBMQProviderError
in_page_toc_min_heading_level: 1
python_api_type: exception
python_api_name: qiskit.providers.ibmq.IBMQProviderError
---

<span id="qiskit-providers-ibmq-ibmqprovidererror" />

# qiskit.providers.ibmq.IBMQProviderError

<Class id="qiskit.providers.ibmq.IBMQProviderError" isDedicatedPage={true} github="https://github.com/qiskit/qiskit-ibmq-provider/tree/stable/0.12/qiskit/providers/ibmq/exceptions.py" signature="IBMQProviderError(*message)" modifiers="exception">
  Errors related to provider handling.

  Set the error message.

  ### with\_traceback

  <Function id="qiskit.providers.ibmq.IBMQProviderError.with_traceback" signature="with_traceback()">
    Exception.with\_traceback(tb) â€“ set self.\_\_traceback\_\_ to tb and return self.
  </Function>
</Class>

